src/libDirectoryService/DirectoryService.cpp

```diff
@@ -175,6 +175,15 @@ bool DirectoryService::ProcessSetPrimary(const vector<unsigned char>& message,
     // Message = [Primary node IP] [Primary node port]
     LOG_MARKER();
 
+    if (m_mediator.m_currentEpochNum > 1)
+    {
+        LOG_GENERAL(WARNING,
```

Comment:
add a comment to deduct score from the reputation manager

Reply:
If a attacker send a lot of bootstrape message to this node, then this node's reputation will decrease a lot.

Reply:
Maybe for consistency with other todo in the codebase, maybe we can replace "Work need to do" with TODO. Thanks